### PR TITLE
enhance documentation

### PR DESCRIPTION
**Pull request by wfeldt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,51 @@
...
+drwxr-xr-x          6 Apr 26 12:27 boot/x86_64/grub2-efi/themes/openSUSE
+drwxr-xr-x         17 Jun 20  2016 EFI
+drwxr-xr-x         88 Nov  9 16:21 EFI/BOOT
+-rwxr-xr-x    1155520 Jun 20  2016 EFI/BOOT/bootx64.efi
+-rw-r--r--       2638 Nov  9 16:21 EFI/BOOT/grub.cfg
+-rw-r--r--     975712 Jun 20  2016 EFI/BOOT/grub.efi
+```
+
+You can see the kernel (`linux`), the `initrd`, boot loader files belonging to `isolinux` and `grub2`, and
+files like `root`, `rescue`, `common` that are squashfs images containing the installation system with
```
Comment (lslezak) [inline README.md]: NP: I'd make "squashfs" word a link pointing to http://www.tldp.org/HOWTO/SquashFS-HOWTO/

File: README.md [hunk 2]
```diff
@@ -1,9 +1,51 @@
...
+the installer [YaST](https://en.opensuse.org/Portal:YaST).
```
Comment (lslezak) [inline README.md]: NP: "the YaST installer" would probably sound better

File: README.md [hunk 3]
```diff
@@ -1,9 +1,51 @@
...
+
+If you are going to work on this project, have a look at the documentaion first:
+
+- [General intro](doc/index.md)
+- [Submitting changes to SUSE Build Service](doc/submitting.md)
+- [Modifying the branding](doc/branding.md)
+- [Config options when generating the images](doc/configoptions.md)
```
Comment (lslezak) [inline README.md]: NP: "for" instead of "when"

File: doc/submitting.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+## Submitting changes to SUSE Build Service
+
+Submissions are managed by a SUSE internal jenkins node in the InstallTools tab.
```
Comment (lslezak) [inline doc/submitting.md]: I'd more links:

```markdown
Submissions are managed by a SUSE internal [Jenkins](https://jenkins.io/) node
in the [InstallTools tab](https://ci.suse.de/view/InstallTools/view/installation-images/).
```

File: doc/submitting.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+Each time a new commit is integrated into the master branch of the repository,
+a new submit request is created in the SUSE Build Service.
```
Comment (lslezak) [inline doc/submitting.md]: Um, which one? Internal, external or both? Please, be more specific...

File: doc/submitting.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+For maintained branches, the package is submitted to a devel project but the final submission
```
Comment (lslezak) [inline doc/submitting.md]: Please add a link:
```markdown
a [devel project](https://build.opensuse.org/package/show/system:install:head/installation-images)
```

File: doc/submitting.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+must be triggered manually.
+
+`*.changes` and version numbers are auto-generated from git commits, you don't have to worry about this.
+
+The spec file is maintained in the Build Service only. If you need to change it, submit to the devel project
+in the build service directly. The current names of the devel projects can be seen in the jenkins logs.
```
Comment (lslezak) [inline doc/submitting.md]: This is a bit tricky as the Jenkins is internal only. That means the external contributors have no chance to see them. Is there any other possibility?

File: doc/submitting.md (hunk 4 above)
Comment (wfeldt) [inline doc/submitting.md]: I was rather avoiding giving away internal machine names. :-(

Comment (lslezak) [inline doc/submitting.md]: I see... But I don't think it is a problem, the machine is not available from outside anyway.

**Pull request merged**